common/src/main/java/com/wynntils/core/features/overlays/BarOverlay.java

```diff
@@ -31,11 +31,16 @@ public abstract class BarOverlay extends DynamicOverlay {
     @RegisterConfig("overlay.wynntils.barOverlay.flip")
     public final Config<Boolean> flip = new Config<>(false);
 
+    @RegisterConfig("overlay.wynntils.barOverlay.animated")
```

Comment:
This is a bit odd config name and function. Maybe `animationTime`?

Reply:
Fixed.